src/ui/components/SignMessageButton/SignMessageButton.tsx

```diff
@@ -91,32 +99,68 @@ export const SignMessageButton = React.forwardRef(function SignMessageButton(
 
   const isLoading =
     personalSignMutation.isLoading || signTypedData_v4Mutation.isLoading;
+  const isSuccess =
+    personalSignMutation.isSuccess || signTypedData_v4Mutation.isSuccess;
+  const isError =
+    personalSignMutation.isError || signTypedData_v4Mutation.isError;
+
+  // there is a small delay after using a holdable button
+  // button should be disabled after successful sign to prevent a duplicating call
+  const disabled = isLoading || isSuccess;
```

Comment:
should this be instead
```suggestion
  const disabled = isLoading || (holdToSign && isSuccess);
```

Reply:
This is fair!